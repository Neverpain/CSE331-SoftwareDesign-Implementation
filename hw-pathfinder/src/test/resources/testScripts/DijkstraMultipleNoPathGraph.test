# Tests Dijkstra's algorithm on an "island" graph will always return
# no paths between any given node within the graph, outputs it all

CreateGraph g

AddNode g A
AddNode g B
AddNode g C
AddNode g D
AddNode g E

FindPath g A B

FindPath g A C

FindPath g C E

FindPath g D B

FindPath g D E