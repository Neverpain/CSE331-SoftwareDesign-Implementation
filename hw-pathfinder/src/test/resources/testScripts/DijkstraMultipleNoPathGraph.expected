# Tests Dijkstra's algorithm on an "island" graph will always return
# no paths between any given node within the graph, outputs it all

created graph g

added node A to g
added node B to g
added node C to g
added node D to g
added node E to g

path from A to B:
no path found

path from A to C:
no path found

path from C to E:
no path found

path from D to B:
no path found

path from D to E:
no path found